React/Views/RCTView.m

```diff
@@ -1563,4 +1564,28 @@ - (BOOL)performDragOperation:(id <NSDraggingInfo>)sender
 }
 #endif // ]TODO(macOS ISS#2323203)
 
+#pragma mark - Keyboard Events
+
+#if TARGET_OS_OSX // TODO: add iOS keyboard event handling
+- (void)keyDown:(NSEvent *)event {
+  if (!self.onKeyDown) {
```

Comment:
Didn’t the MS objc style guide mention preferring to explicitly test against `nil`?

```suggestion
  if (self.onKeyDown == nil) {
```